github/timestamp_test.go

```diff
@@ -13,9 +13,10 @@ import (
 )
 
 const (
-	emptyTimeStr         = `"0001-01-01T00:00:00Z"`
-	referenceTimeStr     = `"2006-01-02T15:04:05Z"`
-	referenceUnixTimeStr = `1136214245`
+	emptyTimeStr               = `"0001-01-01T00:00:00Z"`
+	referenceTimeStr           = `"2006-01-02T15:04:05Z"`
+	referenceTimeStrFractional = `"2006-01-02T15:04:05.000Z"` // This format was returned by the Projects API before October 1, 2017.
```

Comment:
GitHub announcement says:

> Starting on Monday October 2nd, 2017

So, wouldn't it be more accurate to say this?

```diff
-before October 1, 2017.
+before October 2, 2017.
```

At least I parse "before day X" to mean "before and not including day X" rather than "before and also on day X".

But this is extremely insignificant.